engines/director/window.cpp

```diff
@@ -189,6 +189,12 @@ void Window::setStageColor(uint32 stageColor, bool forceReset) {
 	}
 }
 
+void Window::setTitleVisible(bool titleVisible) {
+	_titleVisible = titleVisible;
+	setTitleVisibility(titleVisible);
```

Comment:
Once you make this method override `MacWindow::setTitleVisible` you can replace this line with `MacWindow::setTitleVisible(titleVisible);`